resource/Makefile

```diff
@@ -41,7 +41,7 @@ all: $(GOBINDATA_TARGET)
 debug: $(GOBINDATA_DEBUG_TARGET)
 	rm -f $(GOBINDATA_TARGET)
 
-$(TYPESCRIPT_TARGET): $(TYPESCRIPT_SOURCES)
```

Comment:
the old version didn't depend on the files in the directory, so it would not recompile them when they changed.